### PR TITLE
Text expansion bug in _create_traceable_model

### PR DESCRIPTION
**Pull request by rohankshah04**
#704

### CODE REVIEW

**Comment by pquentin** [comment]
buildkite test this please

**Comment by davidkyle** [comment]
buildkite test this please

**Comment by davidkyle** [comment]
@rohankshah04 do you have an example model and Python snippet you are using to load the mode? Thanks

**Comment by rohankshah04** [comment]
I've downloaded the model via my new eland docker container. it looks something like this:

eland_import_hub_model --url <____> \ --hub-model-id naver/splade-v3-distilbert \ --task-type text_expansion

and then I just run a basic text expansion search to query my sparse vectors.

File: eland/ml/pytorch/transformers.py [hunk 1]
```diff
@@ -77,7 +77,7 @@
     "text_similarity",
 }
 ARCHITECTURE_TO_TASK_TYPE = {
-    "MaskedLM": ["fill_mask", "text_embedding"],
+    "MaskedLM": ["fill_mask", "text_embedding", "text_expansion"],
```
Comment (davidkyle) [inline eland/ml/pytorch/transformers.py:80]: This map is used to infer the task type when the `--task-type` parameter is not passed on the command line. The `MaskedLM` architecture does not mean that the model can be used for text_expansion, for text_expansion the user will have to explicitly set `--task-type text_expansion` 

```suggestion
    "MaskedLM": ["fill_mask", "text_embedding"],
```

**Review by davidkyle — COMMENTED** [review]
Looks good I was able to run the SPLADE model in my local cluster with this change. 

I left one suggestion, CI is failing due to linting issues please run `python -m nox -s format`

**Comment by pquentin** [comment]
buildkite build this please

**Comment by daixque** [comment]
@rohankshah04 I need your change but build is failing due to just formatting issue. I raised below to fix it. Please check it out.
https://github.com/rohankshah04/eland/pull/1

**Comment by daixque** [comment]
I raised #740 for the formatter issue

**Comment by daixque** [comment]
Let me close this since the change in this PR is already merged as #740

**Pull request closed without merging**